examples/web_apps/webxr_hand_demo/js/vr_demo.js

```diff
@@ -175,9 +180,25 @@ export class VRDemo {
 
     this.webXRSession.requestAnimationFrame(this.drawVRScene.bind(this));
 
-    this.physicsStepFunction = setInterval(() => {
-      this.sim.stepWorld(1.0 / 60);
-    }, 1000.0 / 60);
+    const handFilepathsByHandIndex = [
```

Comment:
The reason for this is that I am relying on the renderer to dictate everything to do with the objects files, such as what the stage is, what the object spawn order is, and in this case, what the hand models are. Stuff like the stage and object spawn order MUST be decided by the renderer because it needs to preload them first.

Indeed, the hand models are a different case and they can be done within the worker, but I don't like that it would be inconsistent with how the other objects are loaded. Also, the renderer would still need to preload the hand models, so in any case it needs to reference "hand_l_open" anyways.